pallas_app/packages/openapi_member/doc/MemberBatch.md

```
@@ -0,0 +1,15 @@
+# openapi_member.model.MemberBatch
+
+## Load the model package
+```dart
+import 'package:openapi_member/api.dart';
+```
+
+## Properties
+Name | Type | Description | Notes
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)